service/src/main/resources/db/changesets/20221013_pathfunc.yaml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
merge those two files into one cuz apprently liquibase merging is pretty time costly.